tutorials/messengers/telegram/2_attachments.py

```diff
@@ -0,0 +1,202 @@
+            ("main_flow", "animation_node"): cnd.exact_match(Message("animation")),
+            ("main_flow", "image_node"): cnd.exact_match(Message("image")),
+            ("main_flow", "document_node"): cnd.exact_match(Message("document")),
+        }
+    },
+    "main_flow": {
+        "start_node": {
+            TRANSITIONS: {"intro_node": cnd.exact_match(Message("/start"))},
+        },
+        "intro_node": {
+            RESPONSE: Message(
+                'Type "location", "contact", "poll", "sticker" '
+                '"audio", "video", "animation", "image", '
+                '"document" or to receive a corresponding attachment!'
+            ),
+        },
+        "location_node": {
+            RESPONSE: Message(
+                "Here's your location!",
+                attachments=[Location(**location_data)],
+            ),
+        },
+        "contact_node": {
+            RESPONSE: Message(
+                "Here's your contact!",
+                attachments=[Contact(**contact_data)],
+            ),
+        },
+        "poll_node": {
+            RESPONSE: Message(
+                "Here's your poll!",
+                attachments=[
+                    Poll(
+                        question="What is the poll question?",
+                        options=[
+                            PollOption(text="This one!"),
+                            PollOption(text="Not this one :("),
+                        ],
+                    ),
+                ],
+            ),
+        },
+        "sticker_node": {
+            RESPONSE: Message(
+                "Here's your sticker!",
+                attachments=[Sticker(**sticker_data)],
+            ),
+        },
+        "audio_node": {
+            RESPONSE: Message(
+                "Here's your audio!",
+                attachments=[Audio(**audio_data)],
+            ),
+        },
+        "video_node": {
+            RESPONSE: Message(
+                "Here's your video!",
+                attachments=[Video(**video_data)],
+            ),
+        },
+        "animation_node": {
+            RESPONSE: Message(
+                "Here's your animation!",
+                attachments=[Animation(**animation_data)],
+            ),
+        },
+        "image_node": {
+            RESPONSE: Message(
+                "Here's your image!",
+                attachments=[Image(**image_data)],
+            ),
+        },
+        "document_node": {
+            RESPONSE: Message(
+                "Here's your document!",
+                attachments=[Document(**document_data)],
+            ),
+        },
+        "fallback_node": {
+            RESPONSE: Message(
+                "Unknown attachment type, try again! "
+                'Supported attachments are: "location", '
+                '"contact", "poll", "sticker", "audio", '
+                '"video", "animation", "image" and "document".'
+            ),
+        },
+    }
+}
```

Comment:
should also add a node for showcasing receiving attachments from user (a bot would ask user to send it an attachment and then do something about it (preferably something that involves calling `get_bytes` so maybe it would send back hash of the attachment) + would be nice to incorporate showcase of caching as well).